examples/getting_started/react/README.md

```diff
@@ -0,0 +1,20 @@
+# Getting started
```

Comment:
[Question] Location wise. Wy we do not have simple examples/react?
Any reason to have getting_started folder?

Reply:
I think the idea initially was to have a few smaller examples showcasing the different features. So there would have been a subscriptions folder etc. Since that doesn't seem to be the route we are going anymore, it will make more sense to do as you suggest and change the folder to `/examples/react`